generic_config_updater/gcu_field_operation_validators.conf.json

```diff
@@ -10,11 +10,128 @@
+                "td2": ["Broadcom Limited Device b850", "Broadcom Limited Broadcom BCM56850"],
+                "td3": ["Broadcom Limited Device b870", "Broadcom Inc. and subsidiaries Device b870"]
+            }
+	}
+    },
     "tables": {
         "PFC_WD": {
-            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ]
+            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ],
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "PFCWD enable/disable": {
+                        "fields": [
+                            "restoration_time",
+                            "detection_time",
+                            "action",
+                            "global/poll_interval"
+                        ],
+                        "operations": ["remove", "add", "replace"],
+                        "platforms": {
+                            "spc1": "20181100",
+                            "td2": "20181100",
+                            "th": "20181100",
+                            "th2": "20181100",
+                            "td3": "20181100",
+                            "cisco-8000": "20201200"
+                        }
+                    }
+                }
+            }
+        },
+        "BUFFER_POOL": {
+            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ],
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "Shared/headroom pool size changes": {
+                        "fields": [
+                            "ingress_lossless_pool/xoff",
+                            "ingress_lossless_pool/size",
+                            "egress_lossy_pool/size"
+                        ],
+                        "operations": ["replace"],
+                        "platforms": {
+                            "spc1": "20191100",
+                            "td2": "",
+                            "th": "20221100",
+                            "th2": "20221100",
+                            "td3": "20221100",
+                            "cisco-8000": ""
+                        }
+                    }
+                }
+            }
+        },
+        "BUFFER_PROFILE": {
+            "field_operation_validators": [ "generic_config_updater.field_operation_validators.rdma_config_update_validator" ],
+            "validator_data": {
+                "rdma_config_update_validator": {
+                    "Dynamic threshold tuning": {
+                        "fields": [
+                            "dynamic_th"
+                        ],
+                        "operations": ["replace"],
+                        "platforms": {
+                            "spc1": "20181100",
```

Comment:
Do we still need to check build number less than 20220531? Because GCU support is introduced in 202205.
It by default satisfies this rule.

Reply:
That's true, but I have this information in the conf file for completion and clarity. 

I left the comparison in field_operation_validators.py as-is because this version comparison is always going to be necessary for cases only supported in 202211+. Do you think it's better to not define lower versions in the conf file, and only run this version comparison check if the higher version is defined in the conf file? I figure for this implementation, I would have to add code just to check if the version is defined, which may be an extra unnecessary step.